images/dotcms/ROOT/srv/50-database-config.sh

```diff
@@ -6,7 +6,11 @@ source /srv/utils/config-defaults.sh
 
 
 echo "Database config ...."
-[[ -z "${PROVIDER_DB_DNSNAME}" ]] && PROVIDER_DB_DRIVER="POSTGRES"
+if [[ -z "${PROVIDER_DB_DNSNAME}" ]]; then
+  PROVIDER_DB_DRIVER="H2"
```

Comment:
the default database for testing is postgres in case the env var is not provided

Reply:
Fixed